pkg/registry/servicecatalog/broker/storage.go

```diff
@@ -58,7 +58,7 @@ func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, error) {
 
 // NewStorage creates a new rest.Storage responsible for accessing Instance
 // resources
-func NewStorage(opts generic.RESTOptions) rest.Storage {
+func NewStorage(opts generic.RESTOptions) (brokers, brokersStatus rest.Storage) {
```

Comment:
This done so that people can GET/PUT the status part of the resource independently from the rest of it?